gc/base/ModronAssertions.h

```diff
@@ -28,111 +28,60 @@
  */
 
 #include "omrcfg.h"
-#include "omrcomp.h"
+
 #include "omr.h"
+#include "omrport.h"
 #include "ut_j9mm.h"
 
-#include <assert.h>
-
 #ifdef __cplusplus
 extern "C" {
 #endif /* __cplusplus */
 
-/* Currently, there are startup errors pointed out by these assertions so only enable them on combination spec until they are fixed for everyone */
-#if defined(OMR_GC_DEBUG_ASSERTS)
-
-extern void omrGcDebugAssertionOutput(OMRPortLibrary *portLibrary, OMR_VMThread *omrVMThread, const char *format, ...);
-
-#define Assert_MM_true(arg) \
-	do {\
-		if (!(arg)) {\
-			Assert_MM_true_internal(false && (arg));\
-			assert(false && (arg));\
-		}\
-	} while(0)
-
-#define Assert_MM_false(arg) \
-	do {\
-		if (arg) {\
-			Assert_MM_false_internal(true || (arg));\
-			assert(false && !(arg));\
-		}\
-	} while(0)
+void omrGcDebugAssertionOutput(OMRPortLibrary *portLibrary, OMR_VMThread *omrVMThread, const char *format, ...);
```

Comment:
There were cases where `extern` needed. This is c file, not c++. I am ok to remove it in case it does not create problems with linkage (all platforms)